src/settings.js

```diff
@@ -67,12 +67,14 @@ var ASSERTIONS = 1;
 var RUNTIME_LOGGING = 0;
 
 // Chooses what kind of stack smash checks to emit to generated code:
+// Building with ASSERTIONS=1 (the default) causes STACK_OVERFLOW_CHECK default
```

Comment:
`ASSERTIONS=1` is the default in `-O0`, but no other levels.

Or do you mean default in the sense that `-s ASSERTIONS` sets it to 1?

I'd suggest
```suggestion
// Building with ASSERTIONS=1 (the default in -O0) causes STACK_OVERFLOW_CHECK default
```

Reply:
But `-O0` is also the default... I guess thats what I was trying to convey in `absent any other settings.`   Let try re-phrasing.

Reply:
OK.. how a about this re-wording.